setup/gcp/installer/forseti_installer.py

```diff
@@ -26,6 +26,49 @@
 from util import utils
 
 
+class ForsetiInstructions(object):
+    """Forseti post-setup instructions."""
```

Comment:
this can handle more than just `post-setup`, right?

Reply:
I think so, will phrase it to setup instead of post-setup.